configuration/cluster/gke/composition.yaml

```diff
@@ -11,16 +11,13 @@ spec:
     kind: GKE
   resources:
     - base:
-        apiVersion: container.gcp.crossplane.io/v1beta1
-        kind: GKECluster
+        apiVersion: container.gcp.crossplane.io/v1beta2
+        kind: Cluster
         spec:
           forProvider:
-            initialClusterVersion: "1.16"
+            initialClusterVersion: "1.21"
             location: us-west2
-            masterAuth:
-              # setting this master auth user name enables basic auth so that a client (e.g.,
-              # provider-helm), can connect with the generated kubeconfig from the connection secret
-              username: admin
+            serviceAccount: sample-sa
```

Comment:
Where does this sa coming? Does it needs to be created beforehand?

Reply:
looks like it doesn't :) removed it